package.json

```diff
@@ -25,11 +25,13 @@
   "license": "MIT",
   "dependencies": {
     "md5": "^2.2.1",
-    "webpack": "^4.39.1"
+    "webpack": "^5.0.0"
```

Comment:
This should probably be a peer (for users) and dev (for testing) dependency, not a regular dependency.